helm/templates/workflow-pv-dev.yaml

```diff
@@ -1,12 +1,14 @@
+{{ if eq .Values.ENV "dev" }}
```

Comment:
We'll only apply this PV in dev. In non-dev environments on the ADC k8s cluster, the already-configured CephFS-based PV will be used. Reference on that here: https://github.com/DataONEorg/k8s-cluster/blob/main/storage/storage.md